minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -108,6 +108,19 @@ public MiniAccumuloConfig setZooKeeperStartupTime(long zooKeeperStartupTime) {
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfig setExistingZooKeepers(String existingZooKeepers) {
```

Comment:
The runner doesn't have to use MiniAccumuloConfig, it could be using the MiniAccumuloConfigImpl which would get around that concern. For more context, we are very hesitant about putting new things into our public API. Every method/class in the public API increases our API compatibility debt. If it's not needed for user functionality, we tend to err on the side of not including things to make maintaining our compatibility easier.